packages-content-model/roosterjs-content-model-types/lib/parameter/DOMHelper.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * A helper class to provide DOM access APIs
+ */
+export interface DOMHelper {
```

Comment:
So all Editor API that access DOM should be under this helper?
Like getElementAtCursor in current editor?

Reply:
For every necessary DOM access API, I'd like to put them here, so the API provided by IStandaloneEditor directy will only has access to Content Model. This is to encourage people to use Content Model more, and reduce using DOM access.

For `getElementAtCursor`, can we use `getDOMSelection` instead?

Reply:
Let me check that.